Source/Core/UICommon/UICommon.cpp

```diff
@@ -248,4 +256,63 @@ bool TriggerSTMPowerEvent()
   return true;
 }
 
+#if defined(HAVE_XRANDR) && HAVE_X11
+void EnableScreenSaver(Display* display, Window win, bool enable)
+#else
+void EnableScreenSaver(bool enable)
+#endif
+{
+// Inhibit the screensaver. Depending on the operating system this may also
+// disable low-power states and/or screen dimming.
+
+#ifdef __APPLE__
```

Comment:
Could you merge this with the other `__APPLE__` block?

Reply:
Done.